dlp/snippets/inspect_content.py

```diff
@@ -310,6 +310,102 @@ def inspect_table(
 
 # [END dlp_inspect_table]
 
+
+# [START dlp_inspect_column_values_w_custom_hotwords]
+from typing import Dict, List, Union  # noqa: E402, I100
+
+import google.cloud.dlp  # noqa: F811, E402
+
+
+def inspect_column_values_w_custom_hotwords(
+    project: str,
+    table_data: Dict[str, Union[List[str], List[List[str]]]],
```

Comment:
for clarity, please use 2 distinct variables instead of a dictionary that must contain "header" and "rows".
name suggestions: table_header, and table_rows

Reply:
Thank you for your commented, I have updates accordingly, Please check